nutrition_ai/lib/screens/authenticated/settings.dart

```diff
@@ -90,15 +90,19 @@ class _SettingsScreenState extends State<SettingsScreen> {
               // TODO: Create a new widget for date input
               TextInput(
                 controller: birthdayController,
-                defaultValue: DateFormat('yyyy-mm-dd').format(user.birthday),
+                defaultValue: DateFormat('yyyy-M-dd').format(user.birthday),
                 name: 'Birthday (yyyy-mm-dd)',
               ),
               const SizedBox(height: 20),
               SelectInput(
                 controller: sexController,
-                defaultValue: user.sex.toString(),
+                defaultValue: (user.sex ?? '').isNotEmpty ? user.sex : 'none',
```

Comment:
Is this the same as `user.sex ?? 'none'`?

Reply:
The user sex can never be null that way so I needed it to do it this way.